functions/lib/util.js

```diff
@@ -20,3 +22,25 @@ exports.getObjectSizeInBytes = (object) => {
     const recordBuffer = Buffer.from(JSON.stringify(object));
     return recordBuffer.byteLength;
 };
+exports.getFields = (config) => config.fields ? config.fields.split(',') : [];
+exports.areFieldsUpdated = (config, before, after) => {
+    const fields = exports.getFields(config);
+    logs.debug(`fields: ${fields}`);
+    // If fields are not configured, then execute update record.
+    if (fields.lenth == 0) {
```

Comment:
i think this typo will cause this function to always return false

Reply:
Thanks Peter Cardenas for catching that.  Hook'em!

Reply:
No problem! so cool to see another longhorn